lib/src/super_extensions_base.dart

```diff
@@ -151,6 +151,33 @@ extension StringExtension on String {
   void copyToClipboard() => Clipboard.setData(ClipboardData(text: this));
 }
 
+extension NullableStringExtension on String? {
+  /// Extension method on String? to eliminate the need for a null-check before
+  /// checking if a string is empty, inspired by method of same name from kotlin
+  /// Sample usage:
+  /// ```dart
+  /// String? text = null;
+  /// if (text.isBlank) {
+  ///   // do something
+  /// }
+  /// ```
+  bool get isBlank {
+    if (this != null && this!.isNotEmpty) return false;
+
+    return true;
+  }
+
+  /// Extension method which is an inverse of [isBlank] method
+  /// Sample usage:
+  /// ```dart
+  /// String? text = null;
+  /// if (text.isNotBlank) {
+  ///   // do something
+  /// }
+  /// ```
+  bool get isNotBlank => !isBlank;
```

Comment:
This won't be needed as the above one will return false if it is not blank.

Reply:
Hmm, I added this method so it follows the isEmpty/isNotEmpty dichotomy already native to strings.
Do you still think it's not needed?